src/public/modules/forms/admin/directives/edit-form.client.directive.js

```diff
@@ -442,6 +468,11 @@ function editFormController(
   }
 
   const duplicateField = (fieldToDuplicate) => {
+    if (
+      fieldToDuplicate.myInfo !== undefined &&
+      $scope.numMyInfoFields >= $scope.maxMyInfoFields
+    )
+      return
```

Comment:
what's the UX here if they attempt to duplicate a MyInfo field when they already have 30? should we show some sort of error message (maybe a Toastr)?

Reply:
The button disappears when they are on 30, so this code shouldn't actually trigger.